SITES-10835: Documentation (#26)
AdobeDocs/uix: src/pages/services/aem-cf-editor/api/index.md

    }
}
```

### Get Content Fragment

> You can access data about the Content Fragment that is currently being edited by using API method in an extension:

Reviewer: ```suggestion
You can access data about the Content Fragment that is currently being edited by using the `host.contentFragment.getContentFragment()` method in an extension:
```

> You can access data about the Content Fragment that is currently being edited by using API method in an extension:

Author: We removed "persist" method from "extensibility" API due to a few concerns:
-) bypassing all field validations;
-) exposing low level API access (just on the edge of the backend API calls) that after a release we need to support;
-) unstable the current API version (request structure, PATCH approach).

```js
import { register } from "@adobe/uix-guest";
// ...
const init = async () => {
    const registrationConfig = {
        id: extensionId,
        methods: {
            headerMenu: {
                async getButtons() {
                    return [
                        {
                            id: "get-active-cf",
                            label: "Get Active CF / Canvas",
                            onClick: async () => {
                                // Get Content Fragment
                                const contentFragment = await guestConnection.host.contentFragment.getContentFragment();
                            },
                        },
                    ];
                },
            },
        },
    };
    const guestConnection = await register(registrationConfig);
}
init().catch(console.error)
```

#### Result object

> This `contentFragment` object holds the last received state from AEM instance. It does not contain recent changes from Editor (no edits form Canvas, sidebar or changes on Variations) **until they are successfully saved** in AEM.

Reviewer: ```suggestion
This `contentFragment` object holds the last received state from AEM instance. It does not contain recent changes from the Editor (no edits from the canvas, sidebar or changes to variations) **until they are successfully saved** in AEM.
```
